helm/vmss-prototype/templates/vmss-prototype.yaml

```diff
@@ -10,6 +10,35 @@
 {{- end -}}
 {{- end -}}
 
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kamino
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: node-updater
+rules:
+- apiGroups: [""]
+  resources: ["nodes"]
```

Comment:
You will need more than this as we will need to be able to drain a node which means evicting pods.

Also, there is a pod that we create to do the last bit of cleanup before the node is fully ready to be snapshot.

If you want to find the minimum set, it may take a number of tries to do that.